### ISSUE TITLE
Show the Keyboard Focus

### ISSUE THREAD

**Issue by PeterGerrard** [PeterGerrard/WPF.Accessibility.Testing#9]
Users need to know which object has the keyboard focus so that they can anticipate the effect of their keystrokes. To highlight the keyboard focus, use colors, fonts, or graphics such as rectangles or magnification. To audibly highlight the keyboard focus, change the volume, pitch or tonal quality. +
To avoid confusion, applications should hide all visual focus indicators and dim selections that are located in inactive windows (or panes).
Applications should do the following with keyboard focus:
 - one item should always have keyboard focus
 - keyboard focus should be visible and obvious
- selections and/or focused items should be visually highlighted

https://docs.microsoft.com/en-us/dotnet/framework/ui-automation/accessibility-best-practices#show-the-keyboard-focus
